cesium_app/tests/fixtures.py

```diff
@@ -28,9 +28,15 @@ def create_test_project():
 
 
 @contextmanager
-def create_test_dataset(project):
-    header = pjoin(os.path.dirname(__file__), 'data', 'asas_training_subset_classes.dat')
-    tarball = pjoin(os.path.dirname(__file__), 'data', 'asas_training_subset.tar.gz')
+def create_test_dataset(project, type='class'):
+    if type == 'class':
+        header = pjoin(os.path.dirname(__file__),
```

Comment:
This may not be a safe location to write to on all systems, e.g. if cesium was installed as part of Debian or some other distribution where you don't have write access to the library location.